Runtime/MeshOperations/AppendElements.cs

```diff
@@ -264,19 +264,23 @@ public static ActionResult CreateShapeFromPolygon(this ProBuilderMesh mesh, ILis
                 mesh.sharedVerticesInternal = SharedVertex.GetSharedVerticesWithPositions(vertices);
                 mesh.InvalidateCaches();
 
-                Vector3 nrm = Math.Normal(mesh, mesh.facesInternal[0]);
+                
+                if (extrude != 0.0f )
```

Comment:
is it easy to get it to be 0.0f exactly?

Reply:
Yes it is the constructor initialization of the member of the PolyShape that is passed.